pkg/vendors/mellanox/mellanox.go

```diff
@@ -141,6 +142,20 @@ func (m *mellanoxHelper) GetMellanoxBlueFieldMode(PciAddress string) (BlueFieldM
 	return -1, fmt.Errorf("MellanoxBlueFieldMode(): unknown device status for %s", PciAddress)
 }
 
+func (m *mellanoxHelper) MlxResetFW(pciAddresses []string) error {
+	log.Log.Info("mellanox-plugin resetFW()")
+	for _, pciAddress := range pciAddresses {
```

Comment:
@tobiasgiese are you planning to try and reset all nics provided ? then return errors if they happened ?

Reply:
That was the question here https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/733#discussion_r1694769683
Can implement it